app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -35,46 +34,38 @@ def call
 
       private
 
-      def create_authentication_parameters
-        authentication_parameters
-      end
-
-      def authentication_parameters
-        @authentication_parameters ||= @jwt_configuration.authentication_parameters(@authenticator_input)
-      end
-
       def validate_and_decode_token
         @logger.debug(LogMessages::Authentication::AuthnJwt::VALIDATE_AND_DECODE_TOKEN.new)
-        @authentication_parameters.decoded_token = @jwt_configuration.validate_and_decode_token(@authentication_parameters)
+        @jwt_configuration.validate_and_decode_token
       end
 
       def get_jwt_identity
```

Comment:
instead of this method i would do this:
```
def get_jwt_identity
  # Will be changed when real get identity implemented        
  @logger.debug(LogMessages::Authentication::AuthnJwt::GET_JWT_IDENTITY.new)
  jwt_identity
  @logger.debug(LogMessages::Authentication::AuthnJwt::FOUND_JWT_IDENTITY.new(jwt_identity))
end

def jwt_identity
  @jwt_identity ||= @jwt_configuration.jwt_identity
end
```

and then replace all the `@jwt_identity` in the code to `jwt_identity`

Reply:
Done